src/app/zwiazek/historia-zwiazku/UnionSubsection.tsx

```diff
@@ -0,0 +1,21 @@
+import UnionListItem from "./UnionIconItem";
+import { Subsection } from "./data";
+
+export default function UnionSubsection({ subtitle, paragraph, bulletpoints }: Subsection) {
+  return (
+    <section className="pt-5">
+      <h3 className="pb-4 font-lora text-24 font-bold leading-9 text-greenMain contrast:text-yellowContrast">
+        {subtitle}
+      </h3>
+      <p className="pb-5">{paragraph}</p>
+      {bulletpoints && (
+        <ul>
```

Comment:
Could be a list component, no need for defining item them (also `item` in the component name just because it is used within a list is meh :/ )